src/hash_table.h

```diff
@@ -171,6 +171,19 @@ LIBYANG_API_DECL LY_ERR lyht_find_next_with_collision_cb(struct ly_ht *ht, void
  */
 LIBYANG_API_DECL LY_ERR lyht_insert(struct ly_ht *ht, void *val_p, uint32_t hash, void **match_p);
 
+/**
+ * @brief Insert a value into a hash table, without checking for collision.
```

Comment:
If I understand correctly, this inserts a value without checking whether the value has already been inserted, not without checking for hash collisions. That is a huge difference.

Reply:
Agree, I'll fix the description.